cmd/node/config/config.toml

```diff
@@ -917,6 +917,7 @@
     PeerAuthenticationTimeBetweenSendsWhenErrorInSec = 60   # 1min
     PeerAuthenticationThresholdBetweenSends          = 0.1  # 10%
     HeartbeatTimeBetweenSendsInSec                   = 60   # 1min
+    HeartbeatTimeBetweenSendsDuringBootstrapInSec    = 60   # 1min
```

Comment:
do we need to have this set to 1 minute on the public testnets & mainnet? The original code used `HeartbeatTimeBetweenSendsInSec` which was set to 5 minutes.

Reply:
exactly, this was the reason behind the PR. While testing on public testnet, I realized that 5 min with no feedback would be too much